exampleCourse/questions/element/dragDropBlocks/question.html

```diff
@@ -0,0 +1,108 @@
+  <pl-answer correct="true">5</pl-answer>
+  <pl-answer correct="false">6</pl-answer>
+  <pl-answer correct="true">7</pl-answer>
+  <pl-answer correct="false">8</pl-answer>
+  <pl-answer correct="false">9</pl-answer>
+  <pl-answer correct="false">10</pl-answer>
+</pl-order-blocks>
+
+<hr>
+
+<pl-question-panel>
+<p>
+  <strong>Example 3:</strong>  Implement the Python function <code>my_sum</code>, which takes 2 numbers, <code>first</code> and <code>second</code>,
+  and return their sum. Note that the ordering of instantiation, computation, and returning of the value must be correct.
+  Remember to indent your code! </p>
+<p>
+  This example uses the attribute <code>pl-hide-in-panel answer="true"</code> to hide the correct answer. It
+  also uses <code>permutation-mode="ranking"</code> to control how the correct solution is ordered
+  and <code>indent</code> attribute in <code>pl-answer</code> to define expected code indentation.
+</p>
+</pl-question-panel>
+
+<pl-hide-in-panel answer="true">
+<pl-order-blocks answers-name="python-ordered" shuffle-options="true"  permutation-mode="ranking">
+  <pl-answer correct="true" ranking="1" indent="1">def my_sum(first, second):</pl-answer>
+  <pl-answer correct="true" ranking="2" indent="2">sum = 0</pl-answer>
+  <pl-answer correct="true" ranking="3" indent="2">sum += first</pl-answer>
+  <pl-answer correct="true" ranking="3" indent="2">sum += second</pl-answer>
+  <pl-answer correct="true" ranking="4" indent="2">return sum</pl-answer>
+  <pl-answer correct="false">sum + first</pl-answer>
+  <pl-answer correct="false">sum + second</pl-answer>
+  <pl-answer correct="false">This is an example very long tile to test CSS overflow</pl-answer>
+
+</pl-order-blocks>
+</pl-hide-in-panel>
+
+
+<hr>
+
+
+<pl-question-panel>
+<p>
+  <strong>Example 4:</strong> Let $a, b$ be any even numbers. Prove that $a + b$ is always even.
+</p>
+
+<p>This example uses the attribute <code>permutation-mode="ranking"</code> to control how the correct solution is ordered.
+When the order of a correct block is not relevant,
+ the attribute <code>ranking</code> is not provided. In this example, two of the blocks can be placed at any
+position when constructing the solution. The attribute <code>dropzone-layout="verticalLayout"</code> is used
+to display the input area above the output area.</p>
+</pl-question-panel>
+
+ <pl-order-blocks answers-name="latex_answers" shuffle-options="true" permutation-mode="ranking" dropzone-layout="verticalLayout">
+  <pl-answer correct="true" ranking="1" >Proof by Contrapositive:</pl-answer>
+  <pl-answer correct="true" ranking="2" >Take some $m \in \mathbb{Z}$ </pl-answer>
+  <pl-answer correct="true" ranking="2" >Take some $n \in \mathbb{Z}$ </pl-answer>
+  <pl-answer correct="true" >Assume $m$ is even</pl-answer>
+  <pl-answer correct="true" >Assume $n$ is even</pl-answer>
+  <pl-answer correct="true" ranking="3" >$\exists l \in \mathbb{Z}$ such that $m = 2l$</pl-answer>
```

Comment:
I think I added the `title` text for accessibility purposes. I have renamed the `title` attribute into `data-title` (in case we might need to support screenreaders in the future), so the tooltip should no longer show.